### DISCUSSION TITLE
来聊天吧~ Let's chat!

### DISCUSSION THREAD

**Discussion by luckykeeper** [luckykeeper/LOVE69_renpy_remaster#18] [General]
来聊天吧~ Let's chat!

**Comment by silivers** [comment]
> 来聊天吧~ Let's chat!

大佬接下来准备翻译其他作品吗

**Reply by luckykeeper** [reply]
> > 来聊天吧~ Let's chat!
> 
> 大佬接下来准备翻译其他作品吗

首先，感谢你的关注！😀😊还想着这里不会有人来呢😂

不过近期暂时没有翻译其它作品的计划，最近手头的事情比较多，还得把找工作的问题解决了，u1s1，今年应届生找工作太难了，硕士都来本科这里卷了😭

等日后有机会了，大概会考虑用 Unity （可能性高一些）或是 KiriKiri 把本作的（算是）前作的两作挑一个（或者两个都做）汉化，不过那是后话了

本项目的主要目的还是传授汉化移植的方法，毕竟授人以鱼不如授人以渔，而我个人的精力也实在有限，希望通过本项目能够鼓舞一些同学能够参与到汉化中来，能够帮助到想要入门 Ren'Py 而不知道从何下手的同学找到一种有趣的入门的方法。如果有人能够因此参与到开源汉化移植的话，本项目也算是真正的有价值了吧

如果你对这方面感兴趣的话不妨去康康文档站的文档，有条件的话去找一帮志同道合的朋友（找不到也无妨，可以康康项目的提交记录，项目的大部分内容都是我一个人做的），也来一起体验一把做汉化的感觉，相信我，虽然这个过程比较累，但是一定会非常有趣、非常快乐、非常充实的（对于我的话，算是实现了一个人生目标吧），在这之中，你也将会学到很多知识。我是学文科的，大学的专业和计算机更是差了十万八千里，只要想学，写 Galgame 的脚本并不是一件难事。稍微看了一下你的 Github ，感觉你对这些应该也略懂一二（起码比我学校信管系一堆只会用 CSDN ，没听过 Github 的人强），如果想学习 Ren'Py 的话，遇到问题都可以来这里问我，我知无不答（当然时间长了可能有的部分记不清了，这也是为什么整了个文档站的原因）

如果你对这方面没有兴趣，也不要伤心，虽然我暂时没法做汉化了，但是今年好玩的 Gal 特别多，今年的业界也是挺卷的，有着更多比我汉化质量好多了的作品和更多有意思的作品在等你去欣赏

看你过往的 Issues ，你现在应该刚刚进入大学校园，虽然现在基本上都在封校没有机会体验校外的美好，不过也意味着有了更多时间去发展自己的爱好，好好享受大学生活吧~

祝：一切安好，前程光明，共勉！

**Reply by silivers** [reply]
厉害带佬，被你分析的差不多了(`Δ’)我是今年刚刚大一，专业勉强跟计算机相关吧(网络工程)，目前只会一点点c和linux，等明年有技术了会试试汉化的(相比技术我更担心自己这散装日语)，如果过两年带佬还想做汉化的话可以考虑招我过来打打杂。
另外，应届生就业真的有那么卷吗，我们专业平时都调侃自己出来只能进厂牵网线😂

**Reply by luckykeeper** [reply]
> 厉害带佬，被你分析的差不多了(`Δ’)我是今年刚刚大一，专业勉强跟计算机相关吧(网络工程)，目前只会一点点c和linux，等明年有技术了会试试汉化的(相比技术我更担心自己这散装日语)，如果过两年带佬还想做汉化的话可以考虑招我过来打打杂。 另外，应届生就业真的有那么卷吗，我们专业平时都调侃自己出来只能进厂牵网线😂

散装日语别怕，我也是散装日语😂，善用 Yahoo 搜索和字典，也可以试试 deepL 之类的，效果很不错

Ren'Py 引擎基本上就是 Python ，可以说是简化版的 Python ，既然 C 能搞定，这个一定会很轻松。当然现在还有 advjs 之类的用前端语言写的引擎，应该也比较简单，选择有很多，甚至你可以用 godot （C++）做，纯 c 的话印象里好像没啥引擎。不过基本上都是按照文档搓就完了

就业的话，因为我是要跨专业就业（金融->运维），虽然有相关的实习，但是可能还是比较困难（投了150+份），目前有回应的不多。今年的难度，可以举个栗子让你感受一下，前段时间面试太平人寿（信息科技岗），三轮面试，面试终试的时候寄了，岗位要求本科生，招1个，终试的时候一看名单，小十个硕士，其中还有海归，最厉害的有个南开大学的硕士，最后人把硕士要去了，硕士都来卷本科岗了，能不难嘛😭【顺带一提，这个岗得基层轮岗3-5年，逃不过先去卖保险的命运】~~其实本来是想先拿这个保底的~~

后面应该还能有几个面试，希望能够在秋招拿到 offer ，哪怕是一个也行

**Reply by silivers** [reply]
加油加油，与君共勉

**Comment by Qwter26654300** [comment]
找到了真冬在第二次HS后第四次哼的歌：
德沃夏克的自新大陆第四乐章(Symphony No.9 in E minor, Op.95 "From the New World": IV.Allegro con fuoco)
（原曲蛮有气势的wwww）

**Reply by luckykeeper** [reply]
> 找到了真冬在第二次HS后第四次哼的歌： 德沃夏克的自新大陆第四乐章(Symphony No.9 in E minor, Op.95 "From the New World": IV.Allegro con fuoco) （原曲蛮有气势的wwww）

首先蟹蟹你的反馈~
非常熟悉，找了一下，看一下是不是说的这个
https://github.com/luckykeeper/LOVE69_renpy_remaster/tree/dev-renpy8.0.0#2022-05-09
对应的原文
https://github.com/luckykeeper/LOVE69_renpy_remaster/blob/c1463f43149f7b854c04d0a03319c94f88b609e6/scripts/scene06.rpy#L793-L801
之前已经有人找出来啦，加到了夜间构建版本里面，因为目前dev版没有什么感觉能发一版的东西，所以没有正式发版，感兴趣的话可以康康 dev 分支的 README ，上面有写在正式版之后做了些什么改动，感兴趣的话，可以去夜间构建里面下载体验一下哈~
附今天的夜间构建链接：https://github.com/luckykeeper/LOVE69_renpy_remaster/actions/runs/10150475449

**Reply by Qwter26654300** [reply]
确实是这个（不过两年前就找出来了，惭愧……）
不过大佬还在坚持这个项目真的挺强的

**Reply by luckykeeper** [reply]
也说不上是坚持吧，目前主要写 Go + Flutter ，renpy 很少折腾了，目前这边基本上也只是有人提了问题就改的程度啦，新弄的东西基本没有😂
上班了比较忙，空闲时间少了，推 gal 的时间都少了2333
